src/RcppExports.cpp

```diff
@@ -55,12 +55,12 @@ BEGIN_RCPP
 END_RCPP
 }
 // pz_parse_lat
-Rcpp::NumericVector pz_parse_lat(const Rcpp::CharacterVector& x);
+std::vector<float> pz_parse_lat(std::vector<std::string>& x);
```

Comment:
@AlbanSagouis I generally default to `<double>` rather than `<float>`, although that may not be necessary here, because the main advantage is the numeric limits of `<float>` can often accidentally be reached, and cause undefined behaviour. The limits of `<double>` are much higher (see [here](https://en.cppreference.com/w/cpp/types/numeric_limits) and [the bottom of this page](https://en.cppreference.com/w/cpp/types/climits)). The limits in this case are well defined, so that's not likely to be issue, but there is no performance effect anyway, so maybe worth thinking about?

Reply:
I'm not familiar with `<float>` and used it because we mentioned it orally :s
Sticking to `<double>` sounds good, I'll make the modifications.

Reply:
I tried replacing `<float>` by `<double>` but got several test errors, for example:
```
Failure (test-parse_lon.R:46): parse_lon works: run through test_lons
round(parse_lon(test_lons[i]), 5) (`actual`) not equal to -74.64111 (`expected`).

  `actual`: -2330764414375590775143610959527936
`expected`:                                 -75
```
So this will be for later on a dedicated branch